specification/storage/resource-manager/Microsoft.Storage/stable/2023-01-01/storage.json

```diff
@@ -0,0 +1,5738 @@
+          "description": "An array of strings with maximum 10 blob prefixes to be excluded from the inventory."
+        },
+        "blobTypes": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "An array of predefined enum values. Valid values include blockBlob, appendBlob, pageBlob. Hns accounts does not support pageBlobs. This field is required when definition.objectType property is set to 'Blob'."
+        },
+        "includeBlobVersions": {
+          "type": "boolean",
+          "description": "Includes blob versions in blob inventory when value is set to true. The definition.schemaFields values 'VersionId and IsCurrentVersion' are required if this property is set to true, else they must be excluded."
+        },
+        "includeSnapshots": {
+          "type": "boolean",
+          "description": "Includes blob snapshots in blob inventory when value is set to true. The definition.schemaFields value 'Snapshot' is required if this property is set to true, else it must be excluded."
+        },
+        "includeDeleted": {
+          "type": "boolean",
+          "description": "For 'Container' definition.objectType the definition.schemaFields must include 'Deleted, Version, DeletedTime and RemainingRetentionDays'. For 'Blob' definition.objectType and HNS enabled storage accounts the definition.schemaFields must include 'DeletionId, Deleted, DeletedTime and RemainingRetentionDays' and for Hns disabled accounts the definition.schemaFields must include 'Deleted and RemainingRetentionDays', else it must be excluded."
+        },
+        "creationTime": {
+          "type": "object",
+          "$ref": "#/definitions/BlobInventoryCreationTime",
+          "description": "This property is used to filter objects based on the object creation time"
+        }
+      },
+      "description": "An object that defines the blob inventory rule filter conditions. For 'Blob' definition.objectType all filter properties are applicable, 'blobTypes' is required and others are optional. For 'Container' definition.objectType only prefixMatch is applicable and is optional."
+    },
+    "BlobInventoryCreationTime": {
+      "type": "object",
+      "properties": {
+        "lastNDays": {
+          "type": "integer",
+          "minimum": 1,
+          "maximum": 36500,
+          "format": "int32",
+          "description": "When set the policy filters the objects that are created in the last N days. Where N is an integer value between 1 to 36500."
+        }
+      },
+      "description": "This property defines the creation time based filtering condition. Blob Inventory schema parameter 'Creation-Time' is mandatory with this filter."
+    },
+    "StorageAccountMigration": {
+      "properties": {
+        "id": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Migration Resource Id"
+        },
+        "name": {
+          "type": "string",
```

Comment:
The description seems to be misleading. Can customers input the parameter "name"? Also from the api uri, it seems like the name is always "default" and this parameter is not used

Reply:
If the name should always be "default", you can reference this sample for how to handle it in swagger :
https://github.com/Azure/azure-rest-api-specs/blob/de54b003aee3524343bef30852213db29f55cccd/specification/storage/resource-manager/Microsoft.Storage/stable/2022-09-01/storage.json#L5438

Reply:
Modeled after ManagementPolicyName to fix